pkg/kubelet/types/constants.go

```diff
@@ -20,3 +20,13 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// DefaultSysctlWhitelist returns the default whitelist of sysctls and sysctl patterns (ending in *).
+func DefaultSysctlWhitelist() []string {
+	return []string{
+		"kernel.shm*",
```

Comment:
As mentioned in [this proposal comment](https://github.com/kubernetes/kubernetes/pull/26057/files#r75586140), kindly restrict the whitelist to just `kernel.shm_rmid_forced` for v1.4. Let's discuss whether other need customization in the first place after v1.4.